From f6d4b9d8815699b0cb6aae48fdd8392aafa90145 Mon Sep 17 00:00:00 2001
From: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Date: Tue, 5 Mar 2019 17:54:11 -0800
Subject: [PATCH 0539/1852] drm: xlnx: pl_disp: Pass early callback mode

commit 6f0ab4e111f87c987d2dbbb801c096d1480a39ef from
https://github.com/Xilinx/linux-xlnx.git

This patch passes the early callback mode instead of boolean flag.
This is to be in sync with DMA driver implementation.

Signed-off-by: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_pl_disp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_pl_disp.c b/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
index e0e68c5f6826..cf3f4cbe45b9 100644
--- a/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
+++ b/drivers/gpu/drm/xlnx/xlnx_pl_disp.c
@@ -179,7 +179,7 @@ static void xlnx_pl_disp_plane_enable(struct drm_plane *plane)
 	}
 	desc->callback = xlnx_pl_disp->callback;
 	desc->callback_param = xlnx_pl_disp->callback_param;
-	xilinx_xdma_set_earlycb(xlnx_dma_chan->dma_chan, desc, true);
+	xilinx_xdma_set_earlycb(xlnx_dma_chan->dma_chan, desc, EARLY_CALLBACK);
 
 	if (plane->state->fb->flags == DRM_MODE_FB_ALTERNATE_TOP ||
 	    plane->state->fb->flags == DRM_MODE_FB_ALTERNATE_BOTTOM) {
-- 
2.31.1

